packages/discussions/graphql/resolvers/Discussion/comments.js

```diff
@@ -245,6 +244,29 @@ module.exports = async (discussion, args, context, info) => {
     }
   }
 
+  /* 
+    AUTO = comments are sorted 
+    - by date if the first comment was created in the last 24 hours 
+    - or by votes otherwise 
+    the property resolvedOrderBy is just needed when DiscussionOrder === AUTO
+  */
+  let resolvedOrderBy
+  if (orderBy === 'AUTO') {
+    const sortedComments = [...comments].sort(
+      (a, b) => Date.parse(a.createdAt) - Date.parse(b.createdAt),
+    )
+    if (sortedComments.length > 0) {
+      const firstCommentCreatedAt = Date.parse(sortedComments[0].createdAt)
```

Comment:
Probably no need to parse `createdAt` since Pogi will return a `Date` object or soemthing with a similar signature.